homeassistant/components/rflink.py

```diff
@@ -11,6 +11,7 @@
 import os
 
 import async_timeout
+
 from homeassistant.config import load_yaml_config_file
```

Comment:
Some specific reason for the newline? I think it's usual to group "import" statements separatedly from "from X import Y"s though.

Reply:
Will fix this 👍